packages/gatsby-recipes/src/graphql-server/start-dev-cli-server.js

```diff
@@ -0,0 +1,37 @@
+const { createServiceLock } = require(`gatsby-core-utils/dist/service-lock`)
+const execa = require(`execa`)
+
+const startServer = async () => {
+  // Use 50400 as our port as it's a highly composite number! Meaning it has
+  // more divisors than any smaller positive integer.
+  const port = 50400
```

Comment:
Do we need to `getRandomPort(50400)` here, similar to what we do everywhere else for all behind-the-scenes services?

Reply:
@mxstbr this is just for local development so it should be fine hard-coding it — you start this when working with the CLI so you can see the graphql server logs. Otherwise it's hard to get them.